plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveFailureRecoveryTest.java

```diff
@@ -157,4 +156,14 @@ public void testDeletePartitionWithSubquery()
                     Optional.of("DROP TABLE <table>"));
         }).hasMessageContaining("Deletes must match whole partitions for non-transactional tables");
     }
+
+    @Override
+    protected Session enableDynamicFiltering(boolean enabled)
+    {
+        Session session = super.enableDynamicFiltering(enabled);
+        return Session.builder(session)
+                // Ensure probe side scan wait until DF is collected
+                .setCatalogSessionProperty(session.getCatalog().orElseThrow(), "dynamic_filtering_wait_timeout", "1h")
```

Comment:
After https://github.com/trinodb/trino/pull/11709 lands, `dynamic_filtering_wait_timeout` can go back to being in `BaseFailureRecoveryTest` and this override won't be needed.
fyi @ebyhr